README.md

```diff
@@ -0,0 +1,47 @@
+# OpenAssetIO Test Host
+
+A bare-bones C++ host application with [OpenAssetIO](https://github.com/OpenAssetIO/OpenAssetIO)
+as a dependency, for testing basic integration (linking, headers, etc).
+
+Test targets are created that simply check that C++, C and
+Python components can be imported.
+
+> **Note**
+>
+> This project is intended for use in CI and not used as a reference.
+
+## Getting started
+
+This application can link against OpenAssetIO in two modes, depending on
+the value of the `OPENASSETIOTEST_ENABLE_SUBPROJECT` CMake variable -
+see below.
+
+The usual CMake steps can be used to build and run the tests. Assuming
+the current working directory is at the root of the repository:
+
+```shell
+cmake -S . -B build
+cmake --build build
+ctest --test-dir build
+```
+
+
+### `OPENASSETIOTEST_ENABLE_SUBPROJECT=OFF` (default)
+
+OpenAssetIO will be discovered as a CMake package
+(`find_package(OpenAssetIO)`) and so OpenAssetIO must be pre-built and
+installed in a discoverable location.
```

Comment:
[Consider] It's a tad unclear to me whether this description refers to the behaviour of the above option when it's the default, or when it's ON.

Reply:
Hopefully clarified in 06acf102